### PR TITLE
Fix: links

### PR DESCRIPTION
**Pull request by SimplementeCao**
Update the links to the corresponding addresses

- - -
This change is [Reviewable](https://reviewable.io/reviews/cairo-book/cairo-book/641)

### CODE REVIEW

File: src/ch11-06-macros.md [hunk 1]
```diff
@@ -32,11 +32,11 @@ See [How to Write Tests](./ch10-01-how-to-write-tests.md) page.
 
 ## `format!` macro
 
-See [Printing](./ch11-09-printing.md) page.
+See [Printing](./ch09-01-unrecoverable-errors-with-panic.html#panic-macro) page.
```
Comment (TAdev0) [inline src/ch11-06-macros.md:35]: The link is not correct, and should be:

./ch11-09-printing.html#formatting

**Review by TAdev0 — CHANGES_REQUESTED** [review]

File: src/ch11-06-macros.md (hunk 1 above)
Comment (LeandroCarvajal) [reply src/ch11-06-macros.md:35]: Now I modify it. Thks

Comment (TAdev0) [reply src/ch11-06-macros.md]: perfect!

**Review by TAdev0 — APPROVED** [review]

**Comment by TAdev0** [comment]
lgtm

thanks for the PR @LeandroCarvajal

**Pull request merged**